src/lib.rs

```diff
@@ -64,12 +82,13 @@ pub fn run(args: Args) {
             .capture_index_for_name(capture_name)
             .expect(&format!("Unknown capture name: `{}`", capture_name))
     });
+    let output_mode = get_output_mode(&args);
 
     get_project_file_walker(&*supported_language)
         .into_parallel_iterator()
         .for_each(|project_file_dir_entry| {
-            let mut printer = grep::printer::Standard::new_no_color(io::stdout());
-            let path = project_file_dir_entry.path();
+            let mut printer = get_printer(output_mode);
+            let path = format_relative_path(project_file_dir_entry.path(), output_mode);
```

Comment:
It seems like `.search_path()`/`printer.sink_with_path()` will literally print whatever relative path formatting is in the `Path` that you pass it, I guess the `DirEntry.path()` here includes a leading `./` and that's what we wanted for the existing output format but `--vimgrep` mode appears to strip the leading `./` so I just massage it here based on the output mode (before then passing to `.search_path()`/`printer.sink_with_path()`)